subprojects/gdk_rust/gdk_electrum/src/account.rs

```diff
@@ -806,6 +806,22 @@ pub fn create_tx(
 
     let network = &account.network;
 
+    let fee_rate_sat_kb = match request.fee_rate {
```

Comment:
This could use `Option::get_or_insert_with()`, something like `let fee_rate_sat_kb = request.fee_rate.get_or_insert_with(|| match network.id() { .. })`.

Reply:
That's much cleaner, updated as you suggested